### ISSUE TITLE
ComfyUI in img2img inverts the colors

### ISSUE THREAD

**Issue by John-WL** [ModelSurge/sd-webui-comfyui#123]
This only happens in img2img tab, not in txt2img tab:

![15915-3718227357-cute girl with white hair and blue eyes, serious, portrait,](https://github.com/ModelSurge/sd-webui-comfyui/assets/34081873/a723ff5a-3f8b-4b38-912e-91a5d83dbdb6)

**Comment by ljleb** [comment]
`p_img2img_init()` in lib_comfyui.webui.workflow_patcher is responsible. Will push a fix when I find the cause.

**Comment by ljleb** [comment]
Yep, it's a byte overflow. Apparently * 255 on a byte will invert the bits.
